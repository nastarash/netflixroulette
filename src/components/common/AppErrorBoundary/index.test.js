import React from 'react';
import { AppErrorBoundary } from './index';

describe('<AppErrorBoundary />', () => {
  test('should be rendered correctly', () => {
    const errorBoundary = mount(
      <AppErrorBoundary>
        <div>child 1</div>
        <div>child 2</div>
      </AppErrorBoundary>
    );
    expect(errorBoundary).toMatchSnapshot();
  });
});