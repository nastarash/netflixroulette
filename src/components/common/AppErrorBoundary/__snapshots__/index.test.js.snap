// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppErrorBoundary /> should be rendered correctly 1`] = `
<AppErrorBoundary>
  <div>
    child 1
  </div>
  <div>
    child 2
  </div>
</AppErrorBoundary>
`;
